src/main/webapp/resources/css/style.css

```diff
@@ -0,0 +1,13 @@
+html, body {
+    height: 100vh;
+}
+
+.nav-logo {
+    width: 200px;
+}
+
+.homepage-img-cover {
+    background: url(/resources/images/home-background.jpg);
```

Comment:
Wouldn't it be better if you could use a functionality of show-hide but without putting hard coded img paths inside .css, what if we will need to reuse this class functionality in other places in our web app?

Reply:
This is a specific class for homepage image use. I renamed it with a more verbose name.